integration_tests/data/web/data_url_host.csv

```diff
@@ -0,0 +1,2 @@
+original_url,parsed_url
+www.outdoorvoices.com/collections/tennis/,www.outdoorvoices.com
```

Comment:
@axelazaid can you include other example URLs that I had included in my original query that I sent to you? I'd like to make sure that all of those URLs pass this test :)